calcom/cal.com
# fix: Text for cookie consent checkbox in various languages

## What does this PR do?

As I was working on https://github.com/calcom/cal.com/pull/16350 I just figured that the current wording of the cookie consent checkbox text 
```
I consent to our privacy policy and cookie usage
```
doesn't make sense. It should either be 
```
I consent to this website's privacy policy and cookie usage
``` 
or 
```
You consent to our privacy policy and cookie usage
```
As the text is used in the context of a checkbox, I decided to go with the first variant (perspective from "I"). Could also change it to the latter variant if that's preferred.

**Before (English)**
![image](https://github.com/user-attachments/assets/b0dc30bc-bd5c-4b34-8d11-5e2ee4c70cdb)

**After (English)**
![image](https://github.com/user-attachments/assets/d3ab339c-eaa7-469e-9090-71da4b6866b3)

I fixed it for the following languages:
- English (en) (fluent speaker) - adopted string
- German (de) (Native speaker) - adopted string
- Spanish (es) (fluent speaker) - added missing string
- Estonian (et) (translation engine) - adopted string
- French (fr) (fluent speaker) - adopted string
- Italian (it) (translation engine) - adopted string
- Portuguese (pt) (basic speaker / translation engine) - adopted string
- Portuguese Brazilian (pt-br) (translation engine) - adopted string

In brackets I indicated my knowledge of the language / how I translated it. Feel invited to check especially the translations I modified using a translation engine. Also, I didn't translate Hebrew (he) as I'd have no means to verify if it seems to be accurate or not.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

Open the signup page, e.g. http://localhost:3000/signup and check the cookie consent checkbox text for the following languages.

- English (en)
- German (de)
- Spanish (es)
- Estonian (et)
- French (fr)
- Italian (it)
- Portuguese (pt)
- Portuguese Brazilian (pt-br)
